libpod/container.go

```diff
@@ -411,6 +411,11 @@ func (c *Container) Spec() *spec.Spec {
 	return returnSpec
 }
 
+// SpecFromState is a public wrapper for specFromState
+func (c *Container) SpecFromState() (*spec.Spec, error) {
```

Comment:
I'm debating whether we want to fold this into c.Spec()... it does introduce some strangeness around not returning a consistent spec.

Reply:
If we're going to keep it, rename to RunningSpec. Description: "Returns the OCI spec currently being used to run the container, or the spec the container was created with if no running spec is available"

Reply:
removed